src/AccessibilityInsights.SharedUx/ViewModels/BaseActionViewModel.cs

```diff
@@ -141,13 +141,13 @@ protected virtual void InvokeMethod()
         /// <returns></returns>
         internal static BaseActionViewModel GetActionViewModel(A11yPattern p, MethodInfo m)
         {
-            if (m.ReturnType == typeof(DesktopElement) || m.ReturnType == typeof(List<DesktopElement>))
+            if (m.ReturnType == typeof(DesktopElement) || m.ReturnType == typeof(IList<DesktopElement>))
             {
                 return new ReturnA11yElementsViewModel(p, m);
             }
             else if(m.ReturnType.Name != "IAccessible")
             {
-                if (m.ReturnType != typeof(TextRange) && m.ReturnType != typeof(List<TextRange>))
+                if (m.ReturnType != typeof(TextRange) && m.ReturnType != typeof(IList<TextRange>))
```

Comment:
Changed for [GetSelection](https://github.com/microsoft/axe-windows/pull/553/files#diff-015a2f8ec2f5946ef5f4892f8219fc1d23977a5cfd179f9cd6ddcd1634334b59R53) and [GetVisibleRanges](https://github.com/microsoft/axe-windows/pull/553/files#diff-015a2f8ec2f5946ef5f4892f8219fc1d23977a5cfd179f9cd6ddcd1634334b59R59)